solutions/usaco-1185.mdx

```diff
@@ -0,0 +1,129 @@
+---
+id: usaco-1185
+source: USACO Gold 2022 January
+title: Drought
+author: Ryan Chou
+---
+
+[Official Analysis (C++)](http://usaco.org/current/data/sol_prob1_gold_jan22.html)
+
+## Explanation
+
+First, let's split the problem into two cases, if $N$ is even or if $N$ is odd.
```

Comment:
why? what in the problem would make you think that even & odd cases would be any different?

Reply:
~~the bounds~~